.vscode/launch.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
This should not be included. Please, remove the .vscode/ directory in a new commit.

Reply:
Fixed thank you